mirrord/cli/src/container.rs

```diff
@@ -193,6 +192,41 @@ async fn create_sidecar_intproxy(
             )
         })?;
 
+    if matches!(base_command.runtime(), ContainerRuntime::Docker) {
+        let mut inspect_command = Command::new(&runtime_binary);
+        inspect_command
+            .args(["inspect", &sidecar_container_id])
+            .stdout(Stdio::piped());
+
+        let inspect_output = inspect_command.output().await.map_err(|err| {
+            ContainerError::UnsuccesfulCommandOutput(
+                format_command(&inspect_command),
+                err.to_string(),
+            )
+        })?;
+
+        let inspect =
```

Comment:
I would suggest changing the name from `inspect` to something that says what we're inspecting + `_stdout`.